src/main/java/model/LadderLine.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class LadderLine {
+    private ArrayList<Boolean> points = new ArrayList<>();
```

Comment:
Boolean을 가지는 List 를 구현 잘하셨어요.👍
근데 Boolean을 넣어주는 것도 좋지만 Boolean말고 다른 객체를 만들어 List를 만들어 보면 어떨까요??

```suggestion
    private List<CrossBar> bars = new ArrayList<>();
```

이런식으로 객체를 가지고 있으면 어떨까요??